packages/pigeon/example/app/windows/runner/messages.g.h

```diff
@@ -147,6 +147,7 @@ class MessageFlutterApi {
   void FlutterMethod(const std::string* a_string,
                      std::function<void(const std::string&)>&& on_success,
                      std::function<void(const FlutterError&)>&& on_error);
+  static FlutterError CreateConnectionError(const std::string channel_name);
```

Comment:
This should be private, not public.

In fact ideally it should be just be a helper function in the implementation file with internal linkage, so not part of the class and either `static` or in an anonymous namespace. I think we can do that pretty easily by just putting the generation [here](https://github.com/flutter/packages/blob/main/packages/pigeon/lib/cpp_generator.dart#L640).